release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1340,6 +1340,12 @@ For existing storage accounts created before this update, shared access keys are
 +
 link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 
+* Previously, the image registry was unable to run due to a permissions error in the certificate directory. This issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-38885[*OCPBUGS-38885*])
+
+* Previously, the internal image registry did not correctly authenticate users on clusters configured with external OIDC users because the internal image registry used the `openshift specific user` API, which is not available on clusters configured with external OIDC users. Consequently, it was impossible for users to push or pull images to and from the internal image registry. With this update, the internal image registry uses the `SelfSubjectReview` API and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35335[(*OCPBUGS-35335*)]
+
+* Previously, when enabling `virtualHostedStyle` with `regionEndpoint` set in image registry operator config, the image registry would ignore the virtual hosted style config and would fail to start. This update fixes the issue by using a new upstream distribution configuration, which is force path style, in favor of the downstream only version, which is virtual hosted style. (link:https://issues.redhat.com/browse/OCPBUGS-32710[(*OCPBUGS-32710*)]
```

Comment:
```suggestion
* Previously, when enabling the `virtualHostedStyle` configuration with `regionEndpoint` set in the image registry Operator configuration, the image registry ignored the `virtualHostedStyle` configuration and failed to start. This update fixes the issue by using a new upstream distribution configuration, which uses force path style URLs, in favor of the downstream-only version, which is `virtualHostedStyle`. (link:https://issues.redhat.com/browse/OCPBUGS-32710[*OCPBUGS-32710*]
```
Mmm, I think that's the _more_ correct way to say this. Suggestion, of course.

Reply:
good call! Thank you :)